src/main/scala/core/package.scala

```diff
@@ -0,0 +1,79 @@
+import akka.actor.{Actor, ActorRef, Props}
+import core.database.DatabaseActor
+import core.scheduler.{Basched, BaschedRequest}
+import core.utils.Configuration
+import core.webserver.WebServerActor
+
+package object core {
```

Comment:
Ok, if I understand it correctly. You moved all the current code under `core` so they all could access the same constants?

Reply:
Ja!